modules/setting-up-cpu-manager.adoc

```diff
@@ -183,13 +210,13 @@ Node-Selectors:  cpumanager=true
   │ ├─crio-b5437308f1a574c542bdf08563b865c0345c8f8c0b0a655612c.scope
   │ └─32706 /pause
 ----
-+
-Pods of quality of service (QoS) tier `Guaranteed` are placed within the `kubepods.slice`. Pods of other QoS tiers end up in child `cgroups` of `kubepods`:
+
+. Verify that Pods of quality of service (QoS) tier `Guaranteed` are placed within the `kubepods.slice`. Pods of other QoS tiers end up in child `cgroups` of `kubepods`:
 +
 [source,terminal]
 ----
-# cd /sys/fs/cgroup/cpuset/kubepods.slice/kubepods-pod69c01f8e_6b74_11e9_ac0f_0a2b62178a22.slice/crio-b5437308f1ad1a7db0574c542bdf08563b865c0345c86e9585f8c0b0a655612c.scope
-# for i in `ls cpuset.cpus tasks` ; do echo -n "$i "; cat $i ; done
+# cd /sys/fs/cgroup/kubepods.slice/kubepods-pod69c01f8e_6b74_11e9_ac0f_0a2b62178a22.slice/crio-b5437308f1ad1a7db0574c542bdf08563b865c0345c86e9585f8c0b0a655612c.scope
+# for i in `ls cpuset.cpus cgroup.procs` ; do echo -n "$i "; cat $i ; done
```

Comment:
Great

Reply:
you could use crictl 

```
oc debug node/<node-name>
 chroot /host
 crictl ps | grep cpumanager 
 
```

The above crictl output will give you a hash in the first column and using that you can get the pid and cgroup information 

```
crictl inspect <hash> | grep cgroups 
crictl inspect <hash> | grep pid 

```

or 
```
cat /proc/32706/cgroup (32706 is from the systemctl status you posted above)  which should show something like below:

0::/kubepods.slice/kubepods-pod2e30ba8b_1b50_426f_be2a_0e9b3d835d2f.slice/crio-79d72d3961904e9e2259db15d9bac8bdf0280cb1af982ea6c51c46354ed1ae65.scope

```

Reply:
I don't understand you comments here. Could you please describe the changes should be made directly in documentation?